integration_test/tac_test.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:concordium_wallet/screens/terms_and_conditions/widget.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:concordium_wallet/main.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  // TODO Remove / change this test when we remove temp reset T&C page
```

Comment:
Well I would not expect end-to-end tests to be changed because of widget refactoring. :nerd_face:  
But I'll remove it, I just thought it would be nice to mention that this test is testing stuff that will be removed, so if it breaks people won't be hesitant to cut it :shrug: